TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/api/TimoCloudProxyAPIImplementation.java

```diff
@@ -0,0 +1,14 @@
+package cloud.timo.TimoCloud.bungeecord.api;
+
+import cloud.timo.TimoCloud.api.TimoCloudAPI;
+import cloud.timo.TimoCloud.api.TimoCloudProxyAPI;
+import cloud.timo.TimoCloud.api.objects.ProxyObject;
+import cloud.timo.TimoCloud.velocity.TimoCloudVelocity;
+
+public class TimoCloudProxyAPIImplementation implements TimoCloudProxyAPI {
```

Comment:
Remove this class since `TimoCloudBungeeAPIImplementation` already implements this interface.